### DISCUSSION TITLE
Install with PHP 7

### DISCUSSION THREAD

**Discussion by ghost** [fatihemre/football-standings#5] [Show and tell]
Here is the composer you need to install this application on PHP >= 7.4

Delete the composer.lock file first.

!!!! You can't push pull request after that (because of php8 use) !!!!

```
{
    "name": "fatihemre/football-standings",
    "require": {
        "izniburak/router": "^2.5",
        "twig/twig": "^3.0",
        "symfony/dotenv": "5.4.*",
        "symfony/password-hasher": "5.4.*"
    },
    "autoload": {
        "psr-4": {
            "Standings\\": "app"
        },
        "files": [
            "app/helpers.php"
        ]
    }
}
```

**Comment by fatihemre** [comment]
Why we need PHP 7.0? I am currently using Union Types, Return Types(union), i think i can use Null Safe operators. These features comes with PHP 8

**Comment by ghost** [comment]
@fatihemre  Nope. It's for php > 7.4 and it's just for testing and/or if someone have no php8. Typed vars is introduced since php7.

**Reply by fatihemre** [reply]
Yes, typed vars comes with php 7 but union types, composite return types comes with php 8. You can not use "object|false" return type with php 7

**Comment by ghost** [comment]
yes but it's a minor modification for me.
